### DISCUSSION TITLE
Buildozer problem with gradlew (java version?)

### DISCUSSION THREAD

**Discussion by Fr3d-P** [kivy/kivy-github-discussions#44] [Q&A]
Hi,

When Building a program for Android, I've got this message
running gradlew clean assembleDebug

```
[DEBUG]: 	
[DEBUG]:   	FAILURE: Build failed with an exception.
[DEBUG]:   	
[DEBUG]:   	* What went wrong:
[DEBUG]:   	A problem occurred configuring root project 'Brainbow'.
[DEBUG]:   	> Could not open cp_proj generic class cache for build file '/home/frederic/Kivy_shared/Brain-bow/.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/dists/Brainbow/build.gradle' (/home/frederic/.gradle/caches/8.0.2/scripts/5dvofcs26ca0eny7v4zjxz09u).
[DEBUG]:   	   > BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' Unsupported class file major version 65
```
The problem occured after I installed a new version of Ubuntu (Ubuntu 24.04.1 LTS)
I red on Internet that the last message was from gradle, a problem with  java's version,
My java version is 21.0.4, Buildozer version is 1.5.0

I think that gradlew is automatically installed by Buildozer, and it seems to be an old version?
What are my options to select gradle's version in Buildozer?

**Comment by akshayaurora** [comment]
This is not an issue tracker. Please use this forum only for discussions related to the framework , not for issues related to particular repository
